types/bull-arena/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "bee-queue": "^1.2.3"
```

Comment:
I'm assuming that when this PR updates `@types/bull-arena`, it'll add `bee-queue` as a dependency? The library should support `bee-queue` in the range of `[1.0.0,2.0.0)` and no `2.0.0` exists yet. Should I use a different range here?

Reply:
Probably yes. We usually use `"*"`, but that might be too lax for your purposes.